lib/N3Parser.js

```diff
@@ -774,7 +774,13 @@ N3Parser.prototype = {
 
   // ### `_error` emits an error message through the callback
   _error: function (message, token) {
-    this._callback(new Error(message + ' on line ' + token.line + '.'));
+    var err = new Error(message + ' on line ' + token.line + '.');
+    err.context = {
+      token: token,
+      line: token.line,
+      previousToken: this._lexer._prevToken,
```

Comment:
You mean by adding the prevToken as a parameter in the tokenize callback? (https://github.com/OpenTriply/N3.js/blob/8ccffc54904a3f5a5b669f8f6f434c22b2ac1746/lib/N3Parser.js#L897)
I'm comfortable with just accessing the private variable tbh